www/scripts/codecheckerviewer/ListOfBugs.js

```diff
@@ -57,8 +58,21 @@ function (declare, Deferred, ObjectStore, Store, QueryResults, topic,
         function (reportDataList) {
           if (reportDataList instanceof RequestFailed)
             deferred.reject('Failed to get reports: ' + reportDataList.message);
-          else
+          else {
             deferred.resolve(that._formatItems(reportDataList));
+
+            if (query.reportFilters.length > 1)
```

Comment:
Could we create one common filter hook?
This part is almost the same as the `filterHook` function later.